app/assets/javascripts/external-scripts.js

```diff
@@ -89,15 +93,25 @@ ExternalScripts.try = function() {
         script[funcName].bind(script).apply(script,args);
       }
       else {
-        this.externalScriptError("script " + name + " doesn't respond to  " + funcName);
+        this._externalScriptError("script " + name + " doesn't respond to  " + funcName);
```

Comment:
Do we need to log an error?
I guess we could just check if the plugin implements this method and do nothing if it's missing.
That way we won't force scripts to implement dummy `handeEvent`. I'd change it even here, as it won't be breaking change.